### DISCUSSION TITLE
syntax highlighting for yaml

### DISCUSSION THREAD

**Discussion by ChristianKirchnerIlmenau** [WinMerge/winmerge#1177] [Ideas]
It is possible to implement the syntax highlighting for yaml files?

**Comment by sdottaka** [comment]
I have no immediate plans to implement it at the moment.
The source code for each language of syntax highlighting can be found below. I'm hoping someone will add syntax highlighting for yaml there.

https://github.com/WinMerge/winmerge/tree/master/Externals/crystaledit/editlib/parsers
